llama-index-core/llama_index/core/workflow/context.py

```diff
@@ -90,17 +82,21 @@ def data(self):
 
         Use `get` and `set` instead.
         """
+        msg = "`data` is deprecated, please use the `get` and `set` method to store data into the Context."
+        warnings.warn(msg, DeprecationWarning)
```

Comment:
note: might need a sweep for examples/docs that use `ctx.data` 👀

Reply:
Coming with a different PR, it's a big changeset 🙈